packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -200,15 +200,13 @@ export interface ITreeViewConfiguration<TSchema extends ImplicitFieldSchema = Im
 
 // @alpha @sealed
 export interface JsonArrayNodeSchema extends JsonNodeSchemaBase<NodeKind.Array, "array"> {
-    readonly items: {
```

Comment:
This was an oversight in the initial PR. Should have just leveraged `JsonFieldSchema`.